metis-enrichment/metis-enrichment-client/src/test/java/eu/europeana/enrichment/rest/client/DereferenceClientTest.java

```diff
@@ -0,0 +1,56 @@
+package eu.europeana.enrichment.rest.client;
+
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.ArgumentMatchers.anyString;
+import static org.mockito.Mockito.doReturn;
+import static org.mockito.Mockito.eq;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.spy;
+import static org.mockito.Mockito.times;
+import static org.mockito.Mockito.verify;
+
+import java.net.URI;
+import java.util.ArrayList;
+import java.util.List;
+
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.runners.MockitoJUnitRunner;
+import org.springframework.http.HttpEntity;
+import org.springframework.http.HttpMethod;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.client.RestTemplate;
+
+import eu.europeana.enrichment.api.external.model.EnrichmentBase;
+import eu.europeana.enrichment.api.external.model.EnrichmentResultList;
+import eu.europeana.enrichment.utils.InputValue;
+import org.junit.Assert;
+
+@RunWith(MockitoJUnitRunner.class)
+public class DereferenceClientTest {
+	
+	@Test
+	public void testDereference() {
```

Comment:
See EnrichmentClientTest.testGetByUri() for remarks: this code is almost the same.

Reply:
Fixed.